app/assets/javascripts/shared/components/alert.js

```diff
@@ -0,0 +1,20 @@
+const ALERT_ELEMENT = '#float-alert';
+const TEXT_ALERT_ELEMENT = '#float-alert p';
+const HIDE_TIMEOUT = 5000;
+
+function scheduleHide() {
```

Comment:
What do you mean with "inline this code where it's used"? Can you give an example?